tests/test_sklearn_pls_regression.py

```diff
@@ -0,0 +1,85 @@
+"""
+Tests scikit-learn's PLSRegression.
+"""
+
+import unittest
+import numpy
+from sklearn.cross_decomposition import PLSRegression
+from skl2onnx import convert_sklearn
+from skl2onnx.common.data_types import (
+    FloatTensorType, Int64TensorType, DoubleTensorType
+)
+from skl2onnx.common.data_types import onnx_built_with_ml
+from test_utils import (
+    dump_data_and_model,
+)
+
+
+class TestSklearnPLSRegressionConverters(unittest.TestCase):
+    @unittest.skipIf(not onnx_built_with_ml(),
+                     reason="Requires ONNX-ML extension.")
+    def test_model_pls_regression(self):
+        X = numpy.array([[0., 0., 1.], [1., 0., 0.],
+                         [2., 2., 2.], [2., 5., 4.]],
+                        numpy.float32)
+        Y = numpy.array([[0.1, -0.2], [0.9, 1.1], [6.2, 5.9],
+                         [11.9, 12.3]],
```

Comment:
Can we use https://github.com/onnx/sklearn-onnx/blob/43e9c64380396fe71de7ab6dbb499cd52656134f/tests/test_utils/tests_helper.py#L79 here?